src/main/java/core/basesyntax/service/impl/ConverterDataServiceImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ConverterDataService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ConverterDataServiceImpl implements ConverterDataService {
```

Comment:
I think that it is not necessary to create a separate service for deleting the first element. Let`s do it in FruitServiceImpl